iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/converter/ProjectToOrganizationConverter.java

```diff
@@ -0,0 +1,79 @@
+package eu.dnetlib.iis.wf.importer.converter;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.NavigableMap;
+
+import org.apache.commons.collections.MapUtils;
+import org.apache.hadoop.hbase.client.Result;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+
+import eu.dnetlib.data.proto.OafProtos.Oaf;
+import eu.dnetlib.data.proto.OafProtos.OafRel;
+import eu.dnetlib.iis.importer.schemas.ProjectToOrganization;
+import eu.dnetlib.iis.wf.importer.OafHelper;
+import eu.dnetlib.iis.wf.importer.input.approver.ResultApprover;
+
+/**
+ * {@link ProjectToOrganization} converter.
+ * 
+ * @author mhorst
+ *
+ */
+public class ProjectToOrganizationConverter extends AbstractAvroConverter<Collection<ProjectToOrganization>> {
+
+	/**
+	 * Project-organization relation column family.
+	 */
+	private final byte[] relationColumnFamilyBytes;
+
+	// ------------------------ CONSTRUCTORS --------------------------
+
+	/**
+	 * Default constructor.
+	 * 
+	 * @param resultApprover relation approver
+	 * @param relationColumnFamilyBytes project-organization column family
+	 */
+	public ProjectToOrganizationConverter(ResultApprover resultApprover, byte[] relationColumnFamilyBytes) {
+		super(resultApprover);
+		this.relationColumnFamilyBytes = OafHelper.copyArrayWhenNotNull(relationColumnFamilyBytes);
```

Comment:
`resultApprover` was intended to be optional but in reality it will be always set. I've introduced `Preconditions` checks here disallowing nulls. This way it should be straightforward. Appropriate tests were also prepared.